README.md

```diff
@@ -267,3 +267,16 @@ Maintainers keep a folder with a clean checkout of the code and use [jenv.be](ht
 - write Play Store release notes, include link to forum post
 - upload to Play Store
 - if there was an active beta before release (this can happen with point releases), publish a new beta release to replace the previous one which was disabled by the production release
+- attach APK to previously create Github Release with the name `ODK-Collect-vX.X.X.apk`
```

Comment:
`previously created` not `previously create`. 
And maybe `the` before `APK`? Not that sure because it's still confusing to me sometimes if I should use the or not :)